### ISSUE TITLE
Move roadmap to Github project

### ISSUE THREAD

**Issue by loomchild** [puffinrocks/puffin#21]

**Comment by Jeeppler** [comment]
You did move the roadmap to a GitHub project. However, the roadmap project is empty. What is your roadmap?

**Comment by loomchild** [comment]
No, I didn't move it yet, therefore it's empty. I just noticed this new feature on GitHub and was thinking to use it.

The existing roadmap is on Trello (not updated for a while): https://trello.com/b/ov1cHTtu

**Comment by Jeeppler** [comment]
@loomchild what holds you back in using it? I had a look at the tickets you created in trello.com, but did not really understood the context of those cards.

However, one thing I realized is you are heavily thinking about moving your stuff to Kubernetes. Is this still your plan?

**Comment by loomchild** [comment]
What is holding me back - lack of time (sorry for late reply). 

No, I don't want to use kubernetes for now - it doesn't fit my context I believe (lots of small applications vs one complex application).
